components/LiveAnalytics.tsx

```diff
@@ -30,19 +39,38 @@ const init = async () => {
   onLCP(onWebVitalsReport);
 };
 
-if (document.readyState === 'complete') {
-  init();
-} else {
-  window.addEventListener('load', init);
+  window.onerror = function (message, url, lineNo, columnNo, error) {
```

Comment:
What do you think about adding an event listener to the error event handler, something like this:
```js
window.addEventListener('error', function (...) {...}})
```